linked_list/linked_list.cpp

```diff
@@ -0,0 +1,426 @@
+/*
+
+* C++ Program to Implement Linked List
+
+*/
+
+#include<iostream>
+#include<cstdio>
+#include<cstdlib>
+
+using namespace std;
+
+/*
+
+* Node Declaration
+
+*/
+struct node
```

Comment:
`struct` is dominant in C like coding style. Try to use OOP and use `class`. Also implement the getters and setters for all private members.